src/handlers/int-test-support/helpers/testDataHelper.ts

```diff
@@ -50,53 +46,13 @@ export interface GenerateEventsResult {
   eventId?: string;
 }
 
-export const invokeFilterLambdaAndVerifyEventInS3Bucket = async (
-  payload: EventPayload
-): Promise<GenerateEventsResult> => {
-  const updatedSQSEventPayload = await updateSQSEventPayloadBody(
-    payload,
-    "../../../../integration_tests/payloads/validSQSEventPayload.json"
```

Comment:
What happened to this file? Is the test data living somewhere else now?

Reply:
This was just an SQS-event envelope which was used to create the lambda-event payload for the direct invocation. This is luckily no longer needed now.